uploader.json

```diff
@@ -1822,6 +1822,11 @@
             "pattern": [],
             "symbolic_name": "grub2_grubenv"
         },
+        {
+            "command": "/usr/bin/hammer --interactive 0 --output json compute-resource list",
+            "pattern": [],
+            "symbolic_name": "hammer_compute_resource_list"
+        },
```

Comment:
@pipopopo please move this change to the `commands` sections of the json instead of the `files` section.